package com.terziim.backend.exception;

public class StatusMessage {

    public static String SUCCESS = "Successful";
    public static String INTERNAL_SERVER_ERROR = "Internal server error";
    public static String USER_NOT_FOUND = "User not found in system";
    public static String USER_ALREADY_EXISTS = "User already exists in system";
    public static String ROLE_NOT_FOUND = "Role not found";
    public static String EMAIL_HAS_NOT_SENT = "Email has not sent";
    public static String TOKEN_NOT_FOUND = "Token not found";
    public static String TOKEN_HAS_EXPIRED = "Token has expired";
    public static String USERNAME_OR_PASSWORD_IS_INVALID = "Username or password is invalid";
    public static String REFRESH_TOKEN_IS_NOT_VALID = "Refresh token is not valid";
    public static String REFRESH_TOKEN_NOT_FOUND = "Refresh token not found";
    public static String REFRESH_TOKEN_USER_IS_NOT_INVALID = "This token user is not valid";
    public static String JWT_IS_NOT_VALID = "Access token is not valid";
    public static String JWT_HAS_EXPIRED = "Access token has expired";

    public static final String REQUEST_IS_INVALID = "Request is invalid";

    public static final String PASSWORD_IS_WRONG = "Password is wrong";

}
